apis/config/v1alpha1/projectconfig_types.go

```diff
@@ -7,6 +7,59 @@ import (
 	apiv1alpha1 "github.com/os-observability/tempo-operator/apis/tempo/v1alpha1"
 )
 
+// BuiltInCertManagement is the configuration for the built-in facility to generate and rotate
+// TLS client and serving certificates for all Tempo services and internal clients except
+// for the tempo-gateway.
+type BuiltInCertManagement struct {
+	// CACertValidity defines the total duration of the CA certificate validity.
+	CACertValidity string `json:"caValidity,omitempty"`
```

Comment:
does the type represent duration? Could it be changed to use (golang/k8s) duration?

Reply:
Yes,I'll do the change.